### ISSUE TITLE
GET /paraview/292c5766-fa32-11eb-b91b-eb02855d4603

### ISSUE THREAD

**Issue by jourdain** [Kitware/wslink#75]
**Describe the bug**

GET endpoint to retreive session information does not work. The routes are not properly registered.

**To Reproduce**

```
cd tests/simple
python -m wslink.launcher ./launcher.config
```

1. Connect a client and check its session id.
2. Open a new tag and put in the URL `http://localhost:8080/paraview/__SESSION_ID__

***wslink version:***
v1.0.4

**Comment by jourdain** [comment]
I bet the DELETE call is failing the same way...

**Comment by jourdain** [comment]
@scottwittenburg this could be on your list too but low priority. Let me know if you can't do it.

**Comment by jourdain** [comment]
Should be fixed  [v1.10.0](https://github.com/Kitware/wslink/releases/tag/v1.10.0)
